modules/porous_flow/doc/content/modules/porous_flow/natural_convection.md

```diff
@@ -0,0 +1,94 @@
+# Natural Convection
+
+## Problem statement
+
+This example is based on [!citet](Oldenburg1995OnTD), and [!cite](Elder1967TransientCI). The natural convection of water in a 2D porous tank heated on one part of the bottom is modelled. The schematic is demonstrated in [schematicFig]. The experiment details are elaborated in the referred paper so will not be shown here to avoid repetition. Users can refer to [This page](https://mooseframework.inl.gov/modules/porous_flow/governing_equations.html) for governing equations.
```

Comment:
```suggestion
This example is based on the Elder problem [!cite](Elder1967TransientCI), see [!citet](Oldenburg1995OnTD). The natural convection of water in a 2D porous tank heated on one part of the bottom is modelled. The schematic is demonstrated in [schematicFig]. The experiment details are elaborated in the referred paper so will not be shown here to avoid repetition. The governing equations and associated `Kernels` that describe this problem are provided in detail [here]  (https://mooseframework.inl.gov/modules/porous_flow/governing_equations.html).
```